### PR TITLE
Formatting AWS Networking 

### PR DESCRIPTION
**Pull request by emilyroser**
#### Changes made

- fixing some table formatting
- adding missing sub headings
- adding intro blurbs to resource types

### CODE REVIEW

File: source/includes/aws/_subnetworks.md [hunk 1]
```diff
@@ -101,6 +103,7 @@ curl -X GET \
 <code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/subnetworks/:id</code>
 
 Retrieve a subnetwork in a given [environment](#administration-environments).
+
```
Comment (joelplourde4) [inline source/includes/aws/_subnetworks.md:106]: Ah yes! the famous missing whitespace

File: source/includes/aws/_vpcs.md [hunk 2]
```diff
@@ -11,76 +11,83 @@ curl -X GET \
...
+| `instanceTenancy`<br/>_string_ | The tenancy options for instances launched into the VPC (default, dedicated, host), set as shared by default. |
+| `cidrBlock`<br/>_string_       | IPv4 network range for the VPC in CIDR notation.                                                              |
+| `dhcpOptionsId`<br/>_string_   | The DHCP options set applied to the network configuration of the VPC.                                         |
+| `state`<br/>_string_           | The state of VPC access (pending, available).                                                                 |
+| `isDefault`<br/>_boolean_      | Whether or not the VPC is the default environment VPC.                                                        |
+
 
 <!-------------------- RETRIEVE A VPC -------------------->
 
+#### Retrieva a VPC
```
Comment (joelplourde4) [inline source/includes/aws/_vpcs.md]:
```suggestion
#### Retrieve a VPC
```

File: source/includes/aws/_subnetworks.md (hunk 1 above)
Comment (emilyroser) [reply source/includes/aws/_subnetworks.md:106]: lmaooo yes silent killer

**Review by joelplourde4 — CHANGES_REQUESTED** [review]
A little typo to fix but that's very good!

**Review by joelplourde4 — APPROVED** [review]
LETSSS GOOOOOO

**Pull request merged**